news/2022-08-06-new-featured-artist-darkney.md

```diff
@@ -0,0 +1,61 @@
+---
+layout: post
+title: "New Featured Artist: Darkney"
+date: 2022-08-06 18:00:00 +0000
+---
+
+Step into the light with our newest Featured Artist, **Darkney**!
+
+![](https://assets.ppy.sh/artists/290/header.jpg)
+
+Here at osu!, we're lucky to be the beneficiary of many different types of musical material from a variety of artists. First we had image material, and now courtesy of [**Darkney**](https://osu.ppy.sh/beatmaps/artists/290), we've even moved into [*Cloud Material*](https://osu.ppy.sh/beatmapsets/739583) as well.
```

Comment:
is this reference supposed to be subtle? if not then i suggest linking the map

Reply:
yes, also i don't want to link an artist-unrelated map inside an FA article, most readers will have heard of image material before i'm sure